i18n/events/overview.i18n.js

```diff
@@ -117,16 +117,16 @@ export default genI18nMessages({
             youngInspirers: {
                 title: 'Young Inspirers',
                 description:
-                    'Young Inspirers 是首次籌畫的新活動，旨在於創建一個屬於青少年的學習、交流空間。' +
-                    '讓青少年們透過這個活動去分享自身經歷，並培養多元思考、跨族群合作、系統整合的能力，' +
-                    '交流不同世代間的資訊。',
+                    'Young Inspirers 旨在於創建一個屬於青少年的學習、交流空間。' +
```

Comment:
No need to modify the English part?

Reply:
Thanks for pointing this out. I'll check that.